code/modules/mob/living/simple_animal/hostile/bear.dm

```diff
@@ -26,12 +26,73 @@
 	special_parts = list(/obj/item/animal_part/wolf_tooth)
 	faction = "russian"
 	inherent_mutations = list(MUTATION_EPILEPSY, MUTATION_THICK_FUR, MUTATION_IMBECILE, MUTATION_NERVOUSNESS)
+	var/horror_modifer = TRUE // For-admins to turn this off
+	var/rawr_cooldown = FALSE
 
 /mob/living/simple_animal/hostile/bear/FindTarget()
 	. = ..()
 	if(.)
 		playsound(src, 'sound/effects/creatures/bear.ogg', 100, 1, -3)
 
+/mob/living/simple_animal/hostile/bear/proc/rawr_xd()
+	if(.)
+		if(stat != DEAD)
+			visible_message(SPAN_DANGER("[src] stands up and rawrs!"))
+			playsound(src, 'sound/effects/creatures/bear.ogg', 100, 1, -3)
+			for(var/mob/living/carbon/human/H in range(5,src))
+				if(istype(H))
+					if(prob(100 - H.stats.getStat(STAT_VIG))) //Kinda a hard check-ish but cant stack
+						H.stats.addTempStat(STAT_VIG, -STAT_LEVEL_ADEPT, 30 SECONDS, "fear_of_bear")
+						H.stats.addTempStat(STAT_COG, -STAT_LEVEL_ADEPT, 30 SECONDS, "fear_of_bear")
+						H.stats.addTempStat(STAT_BIO, -STAT_LEVEL_ADEPT, 30 SECONDS, "fear_of_bear")
+						H.stats.addTempStat(STAT_MEC, -STAT_LEVEL_ADEPT, 30 SECONDS, "fear_of_bear")
+						H.stats.addTempStat(STAT_ROB, STAT_LEVEL_ADEPT, 30 SECONDS, "fear_of_bear")
+						H.stats.addTempStat(STAT_TGH, STAT_LEVEL_ADEPT, 30 SECONDS, "fear_of_bear")
+						H.added_movedelay -= 0.5
+						addtimer(CALLBACK(H, /mob/living/carbon/human/proc/clear_movement_delay, 0.5), 60)
+						to_chat(H, SPAN_WARNING("The bears rawrs make the feeling of fight or flight all the more apparent."))
```

Comment:
```suggestion
						to_chat(H, SPAN_WARNING("The [src] 's roar triggers the familiar feeling of flight or fight in you!"))
```

I suggest using `[src]` for the sake of consistency with Mukwahs and other types of bear-parent mobs